library/utils.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import json
+import os
+import urllib2
+
+from django.core.files.base import ContentFile
+from django.utils.translation import ugettext as _
+
+OPENLIBRARY_API_URL = 'https://openlibrary.org/api/books?'
+
+
+def to_unicode(text):
```

Comment:
I believe `django.utils.encoding.smart_text` does [exactly that](https://docs.djangoproject.com/en/1.7/ref/unicode/#conversion-functions), don't you think?